styles.js

```diff
@@ -8,7 +8,7 @@ export default createGlobalStyle`
   }
 
   body {
-    margin: 0;
+    margin: 6%;
```

Comment:
A better solution would be to have padding on your main Element / Container in which everything is rendered. Also think about addressing this later when you are using a Layout Component.

Reply:
Okay. Done. Thanks for your input!